components/dropdown/README.md

```diff
@@ -224,34 +231,35 @@ To make your usage of `d2l-dropdown-more` accessible, use the following property
 | `trap-focus` | Boolean, default: `false` | Optionally render a d2l-focus-trap around the dropdown content |
 | `vertical-offset` | Number or String | Provide custom offset, positive or negative |
 
-**Events:**
-* `d2l-dropdown-open`: dispatched when the dropdown is opened
-* `d2l-dropdown-close`: dispatched when the dropdown is closed
-* `d2l-dropdown-focus-enter`: dispatched when the 'trap-focus' attribute is applied and the focus-trap is entered
+### Events
+| Name | Description |
+|---|---|
+| `d2l-dropdown-open` | Dispatched when the dropdown is opened |
+| `d2l-dropdown-close` | dispatched when the dropdown is closed |
+| `d2l-dropdown-focus-enter` | dispatched when the 'trap-focus' attribute is applied and the focus-trap is entered |
 
-**Slots:**
-* `header`: Sticky container at the top of the dropdown
-* `footer`: Sticky container at the bottom of the dropdown
-* `default`: Anything inside of `d2l-dropdown-content` that isn't in the `header` or `footer` slots appears as regular content
+### Slots
+| Name | Description |
+|---|---|
+| `header` | Sticky container at the top of the dropdown |
+| `footer` | Sticky container at the bottom of the dropdown |
+| `default` | Anything inside of `d2l-dropdown-content` that isn't in the `header` or `footer` slots appears as regular content |
+<!-- docs: stop hidden content -->
 
-**Methods:**
-* `async resize()`: Call if the size of the content changes due to a change in
-a nested component. The nested component may choose to fire a custom event, which
-the component containing the `d2l-dropdown-content` can catch and call this method.
+### Methods
```

Comment:
For a page I previously worked on Jeff had mentioned preferring leaving the methods as bullet points

Reply:
Reverted